File: examples/javascript/Designer1/BrowserFormsApplicationWithJavaApplet1/ApplicationApplet.vb
Imports java.applet
Imports ScriptCoreLib.Java.Extensions

Public NotInheritable Class ApplicationApplet
	Inherits Applet
	Public ReadOnly content As New ApplicationControl()

	Public Overrides Sub init()
		content.AttachTo(Me)
		content.AutoSizeTo(Me)
		Me.EnableVisualStyles()
	End Sub 


End Class

File: examples/java/forms/MulticastExperiment/MulticastExperiment/ApplicationApplet.cs
using java.applet;
using ScriptCoreLib.Java.Extensions;

namespace MulticastExperiment
{
    public sealed class ApplicationApplet : Applet
    {
        public readonly ApplicationControl content = new ApplicationControl();

        public override void init()
        {
            content.AttachTo(this);
            content.AutoSizeTo(this);
            this.EnableVisualStyles();
        }

    }
}
